src/js/fc.js

```diff
@@ -544,6 +544,10 @@ const FC = {
                     result.push('SPEKTRUM SRXL2');
                 }
 
+                if (semver.gte(apiVersion, "1.44.0")) {
```

Comment:
We are using now a new API_VERSION constant here.

Reply:
True, I forgot!

Reply:
Thanks guys, just to confirm, you are suggesting replacing the "1.44.0" with API_VERSION_1_44 right?

Reply:
Yes, that is!